src/param.rs

```diff
@@ -117,6 +117,9 @@ pub enum Param {
 
     /// For MDN-sending job
     MsgId = b'I',
+
+    // So that on sending a new message we can sent the subject to "Re: <last subject>"
```

Comment:
This should probably keep the standard of saying "For Chat: so that ..."  Also the ordering here isn't part of any API I think so you could group it with the other For Chat ones I reckon.

Reply:
Ok.